src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -666,6 +666,13 @@ public override BoundNode DefaultVisit(BoundNode node)
 
         #region Expression Visitors
 
+        public override BoundNode VisitLoweredIsPatternExpression(BoundLoweredIsPatternExpression node)
+        {
+            // Ensure this node won't trigger spilling
+            Debug.Assert(this.VisitList(node.Statements) == node.Statements);
+            return node;
```

Comment:
>return node; [](http://example.com/codeflow?start=12&length=12)

I would prefer we delegate to the base rather than hardcoding the result. Then, we assert that we get BoundLoweredIsPatternExpression node back with the original Statements. And not override for ```!DEBUG```.